packages/suite/src/views/suite/SwitchDevice/DeviceItem/AddWalletButton.tsx

```diff
@@ -45,6 +59,7 @@ export const AddWalletButton = ({
         locks.includes(SUITE.LOCK_TYPE.UI);
 
     const onAddWallet = () => {
+        // @TODO distinguish between standard and passphrase wallet
```

Comment:
forgotten?

Reply:
Not forgotten, it's going to be follow up PR